### PR TITLE
fix monitoring and azure native fence default values

### PR DESCRIPTION
**Pull request by yeoldegrove**
This fixes:
 - a logic error in defining `monitoring_netweaver_targets_ha` #724.
 - set default values for azure native fencing variables

### CODE REVIEW

File: aws/main.tf [hunk 1]
```diff
@@ -136,7 +136,7 @@ module "common_variables" {
   monitoring_drbd_targets_ha          = var.drbd_enabled ? local.drbd_ips : []
   monitoring_drbd_targets_vip         = var.drbd_enabled ? [local.drbd_cluster_vip] : []
   monitoring_netweaver_targets        = var.netweaver_enabled ? local.netweaver_ips : []
-  monitoring_netweaver_targets_ha     = var.netweaver_ha_enabled ? [local.netweaver_ips[0], local.netweaver_ips[1]] : []
+  monitoring_netweaver_targets_ha     = var.netweaver_enabled ? (var.netweaver_ha_enabled ? [local.netweaver_ips[0], local.netweaver_ips[1]] : []) : []
```
Comment (arbulu89) [inline aws/main.tf:139]: I would say the next looks more logic:

```
var.netweaver_enabled && var.netweaver_ha_enabled ? [local.netweaver_ips[0], local.netweaver_ips[1]]: []
```

**Review by arbulu89 — APPROVED** [review]
@yeoldegrove It looks good.
I proposed an alternative solution though ;)

File: aws/main.tf (hunk 1 above)
Comment (yeoldegrove) [reply aws/main.tf]: @arbulu89 your approach is now in 7dbfec8
Good to merge?

Comment (arbulu89) [reply aws/main.tf]: Yes. Test is just in case, maybe I wrote some syntax error hehe

**Comment by rtorrero** [comment]
I can confirm that the issue #724 is solved by this. Thanks!

File: aws/main.tf (hunk 1 above)
Comment (yeoldegrove) [reply aws/main.tf]: As a matter of fact, you had one ;)

**Pull request merged**